setup.py

```diff
@@ -54,7 +54,7 @@
 
 
 # The minimum python version which can be used to run ObsPy
-MIN_PYTHON_VERSION = (3, 6)
+MIN_PYTHON_VERSION = (3, 7)
```

Comment:
just not sure about this one @megies @krischer , but for the rest the PR is clean & easy to review :-)

Reply:
It's fine, but thanks for looking out :-)